addons/fullscreen/addon.json

```
@@ -10,7 +10,7 @@
   "info": [
     {
       "type": "notice",
-      "text": "If you choose to hide the toolbar, remember that you can use the Esc key to exit the project player's full screen mode.",
+      "text": "If you choose to always hide the toolbar, remember that you can use the Esc key to exit the project player's full screen mode.",
```

The setting controls when to _show_ the toolbar, so this might be a better description:
```suggestion
      "text": "If you choose to never show the toolbar, remember that you can use the Esc key to exit the project player's full screen mode.",
```